shopfloor/services/zone_picking.py

```diff
@@ -693,6 +693,7 @@ def _set_move_line_as_done(self, move_line, quantity, package, user=None):
         # the field ``shopfloor_user_id`` is updated with the current user
         move_line.shopfloor_user_id = user or self.env.user
 
+    # flake8: noqa: C901
```

Comment:
?

Reply:
too much "if" in this method.
And it looks a bit complicated to split.

Reply:
I see, can you add a TODO pls?